src/estimators/base.py

```diff
@@ -51,10 +51,18 @@ def f(random_state):
 
 
 class EstimatorBase(object):
-  """An estimator takes a sketch and produces a cardinality estimate.
+  """An estimator takes a sketch and produces a frequency histogram.
+
+  If the array of values produced by an estimator is h[], then h[i] represents
+  the cardinality estimate of items having a frequency of i+1.  h[-1] is the
+  cardinality estimate of items having a frequency of len(h) or greater.
+
+  Note that there is no requirement on the number of frequencies that
+  an estimator should produce.  Thus, an estimator that provides estimates
+  only of cardinality may output an array of length 1.
```

Comment:
I think this is the part that's contradicting the doc, but if this is the case, we can't simply use `EstimatorBase[0]` as cardinality estimate?

Reply:
Yes, you are right.